python/chronos/use-case/AIOps/AIOps_anomaly_detect_unsupervised.ipynb

```diff
@@ -188,7 +188,7 @@
     {
      "data": {
       "text/plain": [
-       "<matplotlib.axes._subplots.AxesSubplot at 0x7f0d227e4400>"
+       "<AxesSubplot:title={'center':'cpu_usage of machine 1932'}, xlabel='time_step'>"
```

Comment:
why is this notebook changed?

Reply:
> why is this notebook changed?

Because we change the input para of `set_params` in https://github.com/intel-analytics/BigDL/pull/8104, we need to fix this notebook.